File: javalin/src/test/java/io/javalin/jte/precompiled/JtemultipleparamsGenerated.java
package io.javalin.jte.precompiled;

public final class JtemultipleparamsGenerated {
    public static final String JTE_NAME = "multiple-params.jte";
    public static final int[] JTE_LINE_INFO = {0, 0, 0, 0, 2, 2, 2, 2, 2, 2, 2, 3};

    public static void render(gg.jte.html.HtmlTemplateOutput jteOutput, gg.jte.html.HtmlInterceptor jteHtmlInterceptor, String one, String two) {
        jteOutput.writeContent("<h1>");
        jteOutput.setContext("h1", null);
        jteOutput.writeUserContent(one);
        jteOutput.writeContent(" ");
        jteOutput.setContext("h1", null);
        jteOutput.writeUserContent(two);
        jteOutput.writeContent("!</h1>\n");
    }

    public static void renderMap(gg.jte.html.HtmlTemplateOutput jteOutput, gg.jte.html.HtmlInterceptor jteHtmlInterceptor, java.util.Map<String, Object> params) {
        String one = (String) params.get("one");
        String two = (String) params.get("two");
        render(jteOutput, jteHtmlInterceptor, one, two);
    }
}

File: javalin/src/test/java/io/javalin/jte/precompiled/kte/JtemultipleparamsGenerated.kt
package io.javalin.jte.precompiled.kte

class JtemultipleparamsGenerated {
    companion object {
        @JvmField
        val JTE_NAME = "kte/multiple-params.kte"
        @JvmField
        val JTE_LINE_INFO = intArrayOf(0, 0, 0, 0, 0, 2, 2, 2, 2, 2, 2, 2, 3)
        @JvmStatic
        fun render(jteOutput: gg.jte.html.HtmlTemplateOutput, jteHtmlInterceptor: gg.jte.html.HtmlInterceptor?, one: String, two: String) {
            jteOutput.writeContent("<h1>")
            jteOutput.setContext("h1", null)
            jteOutput.writeUserContent(one)
            jteOutput.writeContent(" ")
            jteOutput.setContext("h1", null)
            jteOutput.writeUserContent(two)
            jteOutput.writeContent("!</h1>\n")
        }

        @JvmStatic
        fun renderMap(jteOutput: gg.jte.html.HtmlTemplateOutput, jteHtmlInterceptor: gg.jte.html.HtmlInterceptor?, params: Map<String, Any>) {
            val one = params.get("one") as String
            val two = params.get("two") as String
            render(jteOutput, jteHtmlInterceptor, one, two);
        }
    }
}
